docs/README.md

```diff
@@ -16,61 +16,45 @@ The LogDNA agent is a blazingly fast, resource efficient log collection client,
-    * [Upgrading on Kubernetes](#upgrading-on-kubernetes)
-* [Installing](#installing)
-    * [Prerequisites](#prerequisites)
-    * [Installing on Kubernetes](#installing-on-kubernetes)
+* [Managing Deployments](#managing-deployments)
+  * [Installing](#installing)
+  * [Upgrading](#upgrading)
+  * [Uninstalling](#uninstalling)
+  * [More Information](#more-information)
 * [Building](#building)
-    * [Building on Linux](#building-on-linux)
-    * [Building on Docker](#building-on-docker)
+  * [Building on Linux](#building-on-linux)
+  * [Building on Docker](#building-on-docker)
 * [Configuration](#configuration)
-    * [Options](#options)
-    * [Configuring Kubernetes](#configuring-kubernetes)
-    * [Run as Non-Root](#run-as-non-root)
+  * [Options](#options)
+  * [Configuring the Environment](#configuring-the-environment)
+  * [Configuring Lookback](#configuring-lookback)
 
-## Upgrading
+## Managing Deployments
 
-### Preamble
+The agent has been tested for deployment to Kubernetes 1.9+ and OpenShift 4.6+ environments. 
 
-The agent comes with configuration files that allow it to be deployed to a number of environments. Often, these configuration files, once set, do not change whereas the image they use can update. This introduces issues where configuration files can fall behind and not provide the up to date features that come with evolving tools. We aim to make changes in the agent backwards compatible with old configuration files, but recommend regularly ensuring your config is up to date. The agent will try to alert users when it can't use new features.
+### Installing
 
-### Upgrading on Kubernetes
+* [Installing Kubernetes](KUBERNETES.md#installing)
+* [Installing OpenShift](OPENSHIFT.md#installing)
 
-We recommend doing a complete reinstall of the agent by removing the existing configuration and installing the latest one.
+### Upgrading
 
-1. A backup of your existing agent yaml is highly recommended:
-```
-kubectl get ds logdna-agent -o yaml > old-logdna-agent.yaml
-```
-2. Delete the existing agent using the yaml you installed with:
-```
-# 1.x.x
-kubectl delete -f https://raw.githubusercontent.com/logdna/logdna-agent/master/logdna-agent-ds.yaml
-
-# 2.0.x
-kubectl delete -f https://raw.githubusercontent.com/logdna/logdna-agent/master/logdna-agent-v2.yaml
-```
-3. Follow our instructions for [installing the agent on kubernetes](#installing-on-kubernetes).
+* [Upgrading Kubernetes](KUBERNETES.md#upgrading)
+* [Upgrading OpenShift](OPENSHIFT.md#upgrading)
 
-## Installing
+### Uninstalling
 
-### Prerequisites
+* [Uninstalling Kubernetes](KUBERNETES.md#uninstalling)
+* [Uninstalling OpenShift](OPENSHIFT.md#uninstalling)
 
-* A LogDNA Account. Create an account with LogDNA by following our [quick start guide](https://docs.logdna.com/docs/logdna-quick-start-guide).
-* A LogDNA Ingestion Key. You can get your ingestion key at the top of [your account's Add a Log Source page](https://app.logdna.com/pages/add-host).
+### More Information
```

Comment:
Thanks for adding in mention of "non-root" here. 

For the title, maybe "Additional Installation Options"? 

Tho... journald... is that really an install option or a configuration option? I'm not sure why journald gets listed here but not Lookback, for example?

Reply:
I went with your first suggestion of raising the non-root section directly. It does seem like an important feature for cluster maintainers so I do see the case for raising it directly in the landing page documentation.

The configuration section of the Journald feature is environment agnostic and only talks about the feature as it pertains to the agent. If you understand how journald works, you can use that information to do whatever they want with the feature.

The environment specific journald setup instructions talk about how to actually enable journald in their perspective environments. They're actual instructions.